### DISCUSSION TITLE
How to generate only pb go files from proto in milvus project ?

### DISCUSSION THREAD

**Discussion by ArenaSu** [milvus-io/milvus#33344] [Q&A and General discussion] [answered]
When i read milvus source code，i need to exec make cmd to generate pb go files.Is there a cmd to generate only pb go files from proto in milvus project ?

**Comment by yhmo** [comment, marked as answer]
https://github.com/milvus-io/milvus/blob/50bfde92f2922a8a807dfee195acc65272a68e3a/Makefile#L208

make generated-proto
